src/backend/distributed/citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
-default_version = '9.1-1'
+default_version = '9.1-collate'
```

Comment:
This should just be 9.1-1.

Reply:
With unknown timelines I've been having sql scripts be `X.Y-featurename` which can then be renamed when we're closer to merging. This avoids merge conflicts

Reply:
We've switched to adding all changes to a single upgrade script per major release.

Reply:
Tried changing to `9.2-1` but tests need to be updated for new version, so that should be a different PR which is blocking this